components/Embeds/NeverHaveIEverEmbed.tsx

```diff
@@ -107,7 +120,93 @@ const NeverHaveIEverEmbed: FC<MainProps> = ({ replayedRounds }) => {
           </DiscordActionRow>
         </DiscordAttachments>
       </DiscordMessage>
+      {messageType == "vote" && <DiscordMessage
+        profile="wouldyou"
+        author={profiles.wouldyou.author}
+        avatar={profiles.wouldyou.avatar}
+        roleColor={profiles.wouldyou.roleColor}
+        bot={profiles.wouldyou.bot}
+        verified={profiles.wouldyou.verified}
+      >
+        <DiscordReply
+          slot="reply"
+          profile="wouldyou"
+          author={profiles.wouldyou.author}
+          avatar={profiles.wouldyou.avatar}
+          roleColor={profiles.wouldyou.roleColor}
+          bot={profiles.wouldyou.bot}
+          verified={profiles.wouldyou.verified}
+        >
+          <p style={{ whiteSpace: "initial" }}>Click to see command</p>
+        </DiscordReply>
+          <p>You've voted that you <span className="font-bold">{haveDone ? "have" : "have not"} done it</span>.</p>
+          <EphemeralRow dismissClick={() => setMessageType(null)} />
+      </DiscordMessage>}
+      {messageType == "results" && <DiscordMessage
+        profile="wouldyou"
+        author={profiles.wouldyou.author}
+        avatar={profiles.wouldyou.avatar}
+        roleColor={profiles.wouldyou.roleColor}
+        bot={profiles.wouldyou.bot}
+        verified={profiles.wouldyou.verified}
+      >
+        <DiscordReply
+          slot="reply"
+          profile="wouldyou"
+          author={profiles.wouldyou.author}
+          avatar={profiles.wouldyou.avatar}
+          roleColor={profiles.wouldyou.roleColor}
+          bot={profiles.wouldyou.bot}
+          verified={profiles.wouldyou.verified}
+        >
+          <p style={{ whiteSpace: "initial" }}>Click to see command</p>
+        </DiscordReply>
+          <DiscordEmbed
+            slot="embeds"
+            color={haveDone ? "#0091ff" : "#f00404"}
+            image={haveDone == null 
+              ? "https://quickchart.io/chart?c=%7Btype%3A%27outlabeledPie%27%2Cdata%3A%7Blabels%3A%5B%27Have%27%2C%27Have+not%27%5D%2Cdatasets%3A%5B%7BbackgroundColor%3A%5B%27%230091ff%27%2C%27%23f00404%27%5D%2Cdata%3A%5B1%2C1%5D%7D%5D%7D%2Coptions%3A%7Bplugins%3A%7Blegend%3Afalse%2Coutlabels%3A%7Btext%3A%27%25l+%25p%27%2Ccolor%3A%27white%27%2Cstretch%3A35%2Cfont%3A%7Bresizable%3Atrue%2CminSize%3A12%2CmaxSize%3A18%7D%7D%7D%7D%7D&w=750&h=750&ref=qc-js&bkg=%232F3136&f=png&v=2"
```

Comment:
I would actually copy the image instead of embed it by link to avoid third party requests from the user

Reply:
I agree, it would probably load instantly and would be easier to handle.

Reply:
Implemented in [5b0bd6d](https://github.com/Would-You-Bot/website-app/pull/41/commits/5b0bd6db0ce87754a3c78cfd332297f286fd1032)